great_expectations/datasource/data_connector/configured_asset_sql_data_connector.py

```diff
@@ -22,6 +19,16 @@
 
 
 class ConfiguredAssetSqlDataConnector(DataConnector):
+    """
+    A DataConnector that requires explicit listing of Sql tables you want to connect to.
```

Comment:
Perhaps we should use SQL (all capitals in the acronym) when as part of documentation?  Thanks.

Reply:
done